### PR TITLE
Updated README for repo consistency project

### PR DESCRIPTION
**Pull request by paperclypse**
For context:
https://docs.google.com/document/d/1JIog8rJL9MT6_AbIji8VK_RT3mOuIxC2Xb_xQsVksKk/edit#

https://newrelic.atlassian.net/browse/DOC-6121

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,13 +54,18 @@ Once installed, you can get started with a simple program that will create a met
     })
 ```
 
-## Key Concepts
+## Getting Started
+>[Simple steps to start working with the software similar to a "Hello World"]
+
```
Comment (michaelgoin) [inline README.md:59]: Should remove this unused section prior to merge.

**Review by michaelgoin — COMMENTED** [review]
Looks like a good start towards consistency.  Wwe haven't attempted yet for this repo but have an open issue.

We should remove the empty section and placeholder text before merging. When we eventually tackle full consistency, we can determine if we want to add back.

File: README.md (hunk 1 above)
Comment (paperclypse) [reply README.md:59]: Works for me!

**Review by michaelgoin — APPROVED** [review]
👍

**Pull request merged**